ado-pipeline/steps/tf-install.yaml

```diff
@@ -0,0 +1,16 @@
+parameters:
+  - name: terraformVersion
+    type: string
+    default: 1.1.7
```

Comment:
I would remove the default from all of these terraformVersion parameters and have the version stored/passed in from a single location